### ISSUE TITLE
Run ResNext101 on EgoGesture

### ISSUE THREAD

**Issue by dcprime** [ahmetgunduz/Real-time-GesRec#72]
Hi there! I'm a newbie at this (studied the theory, trying to learn how to put it into practice!) and I'm hoping you can help with a simple request.

I'd like to run the EgoGesture set through the ResNext101 pretrained network to see it perform gesture classification. I've downloaded the egogesture_resnext_101_Depth_32.pth pretrained model from your link, and I have recently downloaded the EgoGesture data set. How would I go about running the network on the dataset so that I can see the results?

Thanks for your help!

**Comment by prabinaneupane** [comment]
Were you able to train it?

**Comment by dcprime** [comment]
No, I never could figure out how to run the network from the code here. We ended up using a different network from the PyTorch library instead.

**Comment by Alex-Beh** [comment]
@dcprime I didn't find any clue how to run the network from this code. May I know the alternative network that you use eventually?

**Comment by dcprime** [comment]
We used a 3D version of ResNet-18. You can find it in the PyTorch library here: https://pytorch.org/docs/stable/torchvision/models.html#resnet-3d
